swift/app.py

```diff
@@ -5,21 +5,30 @@
 """
 
 import json
-from typing import Any, Optional, Iterable
+from typing import Any, Dict, Optional, Callable, Iterable
 
 from PyQt5.QtCore import QObject, pyqtSignal, pyqtSlot
 from PyQt5.QtWidgets import QWidget
 
+from swift import swift
+
 class BaseApp(QObject):
     """Base App class that all apps should inherit.
 
     Signals: 
-        broadcastRequested(str, str): A signal for broadcasting to a channel 
-          which contains the target channel name and the message.
-        received(str, str): A signal for receiving a global signal from a channel
-          which contains the source channel name and the message.
-        TODO(kangz12345): docstring for swiftcallRequested and swiftcallReturned.
-          They should be written when the swiftcall protocol is finally determined.
+        broadcastRequested(str, str): The app requested broadcasting to a channel 
```

Comment:
It seems that the past expression can cause a little confusion.
How about including that this signal will be emitted once the app requests broadcasting?

Reply:
Okay, I will make the expression more precise.